mlflow/store/ftp_artifact_repo.py

```diff
@@ -62,89 +62,67 @@ def _size(self, full_file_path):
             ftp.voidcmd('TYPE A')
         return size
 
-    def download_files(self, path, destination):
-        with self.get_ftp_client() as ftp:
-            ftp.cwd(path)
-            if not os.path.isdir(destination):
-                os.makedirs(destination)
-
-            filelist = ftp.nlst()
-
-            for ftp_file in filelist:
-                if self._is_dir(build_path(path, ftp_file)):
-                    self.download_files(build_path(path, ftp_file),
-                                        build_path(destination, ftp_file))
-                else:
-                    with open(os.path.join(destination, ftp_file), "wb") as f:
-                        ftp.retrbinary("RETR "+ftp_file, f)
+    def get_path_module(self):
+        import posixpath
+        return posixpath
```

Comment:
Do we believe that in general posixpath is a better choice than os.path when targeting an NFS server?